omnivoice/utils/i18n.py

```
@@ -0,0 +1,80 @@
+    """
+    global _current_translations, _current_lang
+
+    locales_dir = get_locales_dir()
+
+    if lang is None:
+        try:
+            # Try to get system default locale
+            system_lang, _ = locale.getdefaultlocale()
+            if system_lang:
+                lang = system_lang
+        except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)